src/test/java/com/jcabi/jdbc/JdbcSessionITCase.java

```diff
@@ -91,6 +98,84 @@ public void changesTransactionIsolationLevel() throws Exception {
     }
 
     /**
+     * JdbcSession can run a function (stored procedure) with
+     * output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithOutParam() throws Exception {
+        final DataSource dsrc = JdbcSessionITCase.source();
+        new JdbcSession(dsrc).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS users (name VARCHAR(50))"
+        ).execute().sql("INSERT INTO users (name) VALUES (?)")
+        .set("Jeff Charles").execute().sql(
+            StringUtils.join(
+                "CREATE OR REPLACE FUNCTION getUser(username OUT text)",
+                " AS $$ BEGIN SELECT name INTO username from  users; END;",
+                " $$ LANGUAGE plpgsql;"
+            )
+        ).execute().commit();
+        final Object[] result = new JdbcSession(dsrc)
+            .sql("{call getUser(?)}")
+            .prepare(
+                new Preparation() {
+                    @Override
+                    public void prepare(
+                        final PreparedStatement stmt) throws SQLException {
+                        ((CallableStatement) stmt)
+                            .registerOutParameter(1, Types.VARCHAR);
+                    }
+                }
+             )
+            .call(new StoredProcedureOutcome<Object[]>(1));
+        MatcherAssert.assertThat(result.length, Matchers.is(1));
+        MatcherAssert.assertThat(
+            result[0].toString(),
+            Matchers.containsString("Charles")
+        );
+    }
+
+    /**
+     * JdbcSession can run a function (stored procedure) with
+     * input and output parameters.
+     * @throws Exception If something goes wrong
+     */
+    @Test
+    public void callsFunctionWithInOutParam() throws Exception {
+        final DataSource dsrc = JdbcSessionITCase.source();
+        new JdbcSession(dsrc).autocommit(false).sql(
+            "CREATE TABLE IF NOT EXISTS usersid (id INTEGER, name VARCHAR(50))"
+        ).execute().sql("INSERT INTO usersid (id, name) VALUES (?, ?)")
+        .set(1).set("Marco Polo").execute().sql(
+            StringUtils.join(
+                "CREATE OR REPLACE FUNCTION getUserById(uid IN INTEGER,",
+                " usrnm OUT text) AS $$ BEGIN",
```

Comment:
@amihaiemil thanks for figuring it out, maybe we could drop a very short note about their official name (so that they become googleable) in Javadoc or comment?